### ISSUE TITLE
MultiGPU finetuning

### ISSUE THREAD

**Issue by siyiding1216** [chuanqi305/MobileNet-SSD#91]
Hi, 
Did you try using multi-GPU for training?
I am hoping to finetune your model with 1/10 of the learning rate. 
Wondering what is the change needed if using 8 GPU.

**Comment by madhavajay** [comment]
Has anyone had any luck with Multi GPU training?
I installed NCCL 2 but the speed for 10 iterations is 20 seconds on 1 P100 and the same on 2 P100's and yet it looks like both are being used:
[0] Tesla P100-PCIE-16GB | 50'C,  98 % |  6434 / 16280 MB | madhavajay(6341M) root(82M)
[1] Tesla P100-PCIE-16GB | 53'C,  98 % |  6171 / 16280 MB | madhavajay(6161M)

Can anyone help?
